src/Header/SetCookie.php

```diff
@@ -337,7 +337,7 @@ public function getFieldValue()
         }
 
         $sameSite = $this->getSameSite();
-        if ($sameSite !== null && in_array($sameSite, self::SAME_SITE_ALLOWED_VALUES, true)) {
+        if ($sameSite !== null && array_key_exists(strtolower($sameSite), self::SAME_SITE_ALLOWED_VALUES)) {
```

Comment:
coding style: used functions must be imported with `use function` statements (under `use` block, separated by blank line)
Import just those you touched, that will be enough.